azure/azure-release-pipeline.yml

```diff
@@ -187,8 +233,14 @@ stages:
   dependsOn: 
     - DeployRef
   variables:
```

Comment:
The SERVICE_BASE_PATH for this environment needs to be different -- `oauth2-no-smartcard`